src/screens/People/People.js

```diff
@@ -409,6 +383,31 @@ class PeopleScreen extends React.Component<Props, State> {
     }
   };
 
+  renderEmptyState = ({ inviteTitle, esTitle, esBody }) => {
+    const { referralsFeatureEnabled } = this.props;
+
+    return (
+      <Wrapper fullScreen style={{ marginTop: 8, marginBottom: spacing.large }}>
+        {!referralsFeatureEnabled
+          ? (
+            <EmptyStateWrapper>
+              <EmptyStateParagraph
+                title={esTitle}
+                bodyText={esBody}
+              />
+            </EmptyStateWrapper>
+          )
+          : (
+            <InviteBanner
+              title={inviteTitle}
+              onInvitePress={this.handleInvitePress}
+            />
+          )
```

Comment:
idk I would move this ternary handling before return and add the result var here instead, but as this piece is temporary (referrals going to be enabled anyway) – not sure if it makes sense 🤔

Reply:
Yeah, didn't want to repeat Wrapper twice :)) and yeah, this flag will be removed eventually so there will be just one type of empty state :)